lib/language_pack/rails2.rb

```diff
@@ -45,7 +45,8 @@ def default_process_types
 
   def compile
     instrument "rails2.compile" do
-      install_plugins
+      # XXX(dmauldin): we do not wish for you to stomp our logger object, thanks
+      # install_plugins
```

Comment:
I had expected a change to this line (https://github.com/mtnsat/ruby-buildpack/blob/master/lib/language_pack/rails3.rb#L47) to avoid only the logging plugin. do we really want to remove all plugins?